mizar/obj/bouncer.py

```diff
@@ -135,10 +135,17 @@ def load_transit_xdp_pipeline_stage(self):
     def update_eps(self, eps):
         for ep in eps:
             self.eps[ep.name] = ep
-            if ep.type == OBJ_DEFAULTS.ep_type_simple:
+            if ep.droplet_obj:
                 self._update_simple_ep(ep)
             if ep.type == OBJ_DEFAULTS.ep_type_scaled:
                 self._update_scaled_ep(ep)
+            if ep.type == OBJ_DEFAULTS.ep_type_gateway:
+                self.update_gw_ep(ep)
+
+    def update_gw_ep(self, ep):
+        logger.info("Update gateway endpoint")
+        ep.set_backends([self.ip])
```

Comment:
The gw does not have a backend, why this statement is needed?

Reply:
We talked about setting the remote_ip of the gateway to be bouncer ip that the RPC is being called on.

Reply:
Yeah, I forgot that.